karma.conf.js

```diff
@@ -40,7 +40,7 @@ module.exports = function(config) {
         autoWatch: true,
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-        browsers: ['Chrome', 'Firefox'],
+        browsers: ['Firefox'],
```

Comment:
this concerns tests you run locally because Travis already runs them only with Firefox.
I did that in order to save some of my memory. Because usually, after 15min, the command exits with an error related to memory.
Do you prefer to keep it ? I could add a parameter to the command on my side if you prefer.